docs/src/components/interconnect-tabs.tsx

```diff
@@ -6,25 +6,26 @@ import { groupedMetadata, InterconnectDetails } from "./hardware-utils";
 
 interface InterconnectTabsProps {
   items: HardwareMetadata[];
+  gpio: Boolean;
 }
 
-function mapInterconnect(interconnect: Interconnect) {
+function mapInterconnect(interconnect: Interconnect, gpio: Boolean) {
   let content = require(`@site/src/data/interconnects/${interconnect.id}/design_guideline.md`);
   let imageUrl = require(`@site/docs/assets/interconnects/${interconnect.id}/pinout.png`);
-
   return (
     <TabItem value={interconnect.id} key={interconnect.id}>
       <img src={imageUrl.default} />
-
-      <content.default />
-
+      {gpio && <content.default />}
       {interconnect.node_labels && (
         <>
+          {!gpio && <br></br>}
           The following node labels are available:
           <ul>
-            <li>
-              GPIO: <code>&{interconnect.node_labels.gpio}</code>
-            </li>
+            {gpio && (
```

Comment:
Can we document only GPIO in the new shield guide? I don't think other node labels are necessary in the kscan context, especially since the pinctrl docs will have them.